x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -546,6 +561,14 @@ export const CreatePackagePolicySinglePage: CreatePackagePolicyParams = ({
             <EuiSpacer size="m" />
           </>
         )}
+        {isConfirmModalOpen ? (
+          <UnprivilegedConfirmModal
```

Comment:
above in this file, there is another modal nested under `formState === 'CONFIRM'`

I'm concerned about potential conflicts with that modal since this one is outside of that conditional. can `formState` be part of the condition for opening this modal?

btw, if we continue to use `isConfirmModalOpen`, it should probably be renamed because there are multiple confirm modals in this flow and this is only for the unprivileged modal